include/flatbuffers/grpc.h

```diff
@@ -321,6 +321,17 @@ template<class T> class SerializationTraits<flatbuffers::grpc::Message<T>> {
     }
 #endif
   }
+
+  static grpc::Status Deserialize(ByteBuffer *buffer,
+                                  flatbuffers::grpc::Message<T> *msg) {
+    return Deserialize(buffer->get_buffer(), msg);
+  }
+
+  static grpc::Status Deserialize(ByteBuffer::ByteBufferPointer p,
+                                  flatbuffers::grpc::Message<T> *msg) {
+    return Deserialize((grpc_byte_buffer*)(p) , msg);
```

Comment:
This cast seems funny.. if `ByteBuffer::ByteBufferPointer` is the new way they're writing it, maybe make the above function also take one? Or inline the old function?

Reply:
I just realized this need support from gRPC code so that I submitted two PRs: https://github.com/grpc/grpc/pull/17603 and https://github.com/grpc/grpc/pull/17605. The problem is ByteBuffer doesn't provide a way to get its underlying buffer. To make things working in an efficient way, either of the two PRs need to be approved...

Reply:
Thanks, I'll ping someone to review it.